packages/manager/modules/core/src/redirection/redirection.constants.js

```diff
@@ -79,8 +79,8 @@ export default {
     expressOrder: {
       ASIA: 'https://ca.ovh.com/asia/order/express/',
       AU: 'https://ca.ovh.com/au/order/express/',
-      CA: 'https://ca.ovh.com/en/order/express/',
-      QC: 'https://ca.ovh.com/fr/order/express/',
+      CA: 'https://ca.ovh.com/en/order/express/#/express/review',
+      QC: 'https://ca.ovh.com/fr/order/express/#/express/review',
```

Comment:
Does it makes sense here to make the same changes for other regions? It looks like they redirect also to the same hash